.gitignore

```diff
@@ -37,4 +37,6 @@ exports/
 *.log
 
 # vscode
-.vscode
\ No newline at end of file
+.vscode
+
+/exports/
```

Comment:
`exports/` can already be found on line 34.

Reply:
Noted. However, it was not working for my Directory.

Reply:
That's surprising. It's formatted according to the docs: https://git-scm.com/docs/gitignore